### PR TITLE
DOCSUP-33856 Updated docstrings in collector.py

### PR DESCRIPTION
**Pull request by lehasm**
## Description
Updated docstrings in collector.py

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation and examples improvement (changes affected documentation and/or examples)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/client/collectors.py [hunk 1]
```diff
@@ -62,34 +62,36 @@ def validate_condition(self, conditions: List[RuleCondition]):
...
 class AcceptanceRate(CollectorConfig, spec_value=CollectorConfig.Type.ACCEPTANCE_RATE):
-    """Results of checking the answers of the Toloker.
+    """Counts accepted and rejected Toloker's assignments.
 
-    If non-automatic acceptance (assignment review) is set in the pool, add a rule to:
-    - Set the Toloker's skill based on their responses.
-    - Block access for Tolokers who give incorrect responses.
+    If non-automatic acceptance is set in the pool, you may use this collector to:
+    - Set a Toloker's skill.
+    - Block access for Tolokers with too many incorrect responses.
```
Comment (alexdrydew) [inline src/client/collectors.py:69]:
```suggestion
    - Block access for Tolokers with too many rejected responses.
```
I guess "incorrect" is a bit misleading as the collector does not help you decide which responses are truly incorrect.

File: src/client/collectors.py [hunk 2]
```diff
@@ -151,27 +153,24 @@ class AnswerCount(CollectorConfig, spec_value=CollectorConfig.Type.ANSWER_COUNT)
...
+    - To reassign rejected task suite to other Tolokers increase
+    the overlap of the task suite. It is essential if the default overlap value is 1.
+    - You accept an assignment and don't need to collect more responses for that task suite. To save money stop assigning the task suite.
 
-    Used with conditions:
-    * PendingAssignmentsCount - Number of Assignments pending checking.
-    * AcceptedAssignmentsCount - How many times this assignment was accepted.
-    * RejectedAssignmentsCount - How many times this assignment was rejected.
-    * AssessmentEvent - Assessment of the assignment changes its status to the specified one.
+    The collector can be used with actions:
```
Comment (alexdrydew) [inline src/client/collectors.py:163]:
```suggestion
    The collector can be used with conditions:
```

File: src/client/collectors.py [hunk 3]
```diff
@@ -237,32 +237,34 @@ class Parameters(CollectorConfig.Parameters):
...
+    * [SetSkillFromOutputField](toloka.client.actions.SetSkillFromOutputField.md) sets Toloker's skill value using an output field.
 
     Attributes:
-        parameters.history_size: The number of times the Toloker was shown a captcha recently.
+        parameters.history_size: The maximum number of recent captchas used to calculate the statistics.
+            If `history_size` is omitted, all captchas are counted.
 
     Example:
-        How to ban a Toloker in this project if he mistakes in captcha.
+        The example shows how to block Toloker's access to the project for 15 days if they solve 60% of captchas or less.
```
Comment (alexdrydew) [inline src/client/collectors.py:259]: Should we add "if a Toloker completed captcha at least 3 times"?

**Review by alexdrydew — CHANGES_REQUESTED** [review]

File: src/client/collectors.py (hunk 3 above)
Comment (lehasm) [reply src/client/collectors.py:259]: Added

File: src/client/collectors.py (hunk 2 above)
Comment (lehasm) [reply src/client/collectors.py]: Fixed there and similar parts.

**Review by alexdrydew — APPROVED** [review]

**Pull request merged**